web/src/server_events_dispatch.js

```diff
@@ -215,16 +215,20 @@ export function dispatch_normal_event(event) {
             const realm_settings = {
                 allow_message_editing: noop,
                 avatar_changes_disabled: settings_account.update_avatar_change_display,
+                can_access_all_users_group: noop,
                 can_add_custom_emoji_group: noop,
                 can_add_subscribers_group: noop,
                 can_create_bots_group: noop,
                 can_create_groups: noop,
                 can_create_private_channel_group: noop,
                 can_create_public_channel_group: noop,
+                can_create_web_public_channel_group: noop,
                 can_create_write_only_bots_group: noop,
                 can_delete_any_message_group: noop,
                 can_delete_own_message_group: noop,
+                can_invite_users_group: noop,
```

Comment:
Maybe we can refactor this to just not have to add each setting in the list with `noop`.

Reply:
Yes, I think that'd be a good idea. This is too fragile.